nautobot/core/celery.py

```diff
@@ -93,3 +94,14 @@ def _loads(obj):
 
 # Register the custom serialization type
 register("nautobot_json", _dumps, _loads, content_type="application/x-nautobot-json", content_encoding="utf-8")
+
+
+#
+# nautobot_task
+#
+# By exposing `shared_task` within our own namespace, we leave the door open to
+# extending and expanding the useage and meaning of shared_task without having
```

Comment:
```suggestion
# extending and expanding the usage and meaning of shared_task without having
```

Reply:
This also opens the door for us to extend the base task class used by `shared_task` and make `nautobot_task` use that transparently.